templates/views/account/register.pug

```diff
@@ -37,4 +37,5 @@ block content
 
 block js
   script(src='/js/account.min.js')
+  script(src='/js/validation.js')
```

Comment:
Please add the path for the validation.js file to `grunt/concat.js` and remove it from the .pug file. Else it'll throw a 404 error.

Reply:
thank you for pointing out the issue.
recent changes should resolve the issue